projects/angular-auth-oidc-client/src/lib/storage/browser-storage.service.ts

```diff
@@ -14,32 +14,40 @@ export class BrowserStorageService {
       return false;
     }
 
-    const item = this.getStorage(configId)?.read(key);
+    const storage = this.getStorage(configId);
+
+    if (!storage) {
+      this.loggerService.logDebug(configId, `Wanted to read config for '${configId}' but Storage was falsy`);
+
+      return null;
+    }
+
+    const storedConfig = storage.read(configId);
 
-    if (!item) {
+    if (!storedConfig) {
       return null;
     }
 
-    return JSON.parse(item);
+    return JSON.parse(storedConfig);
   }
 
-  write(key: string, value: any, configId: string): boolean {
+  write(value: any, configId: string): boolean {
     if (!this.hasStorage()) {
-      this.loggerService.logDebug(configId, `Wanted to write '${key}/${value}' but Storage was falsy`);
+      this.loggerService.logDebug(configId, `Wanted to write '${value}' but Storage was falsy`);
 
       return false;
     }
 
     const storage = this.getStorage(configId);
     if (!storage) {
-      this.loggerService.logDebug(configId, `Wanted to write '${key}/${value}' but Storage was falsy`);
+      this.loggerService.logDebug(configId, `Wanted to write '${value}' but Storage was falsy`);
 
       return false;
     }
 
     value = value || null;
 
-    storage.write(`${key}`, JSON.stringify(value));
+    storage.write(`${configId}`, JSON.stringify(value));
```

Comment:
Is there a reason why this is not just like this?

```suggestion
    storage.write(configId, JSON.stringify(value));
```

Reply:
Hey, thanks for the review. The PR is still in WIP and I wanted to look at some things anyway. But this is a nice catch which I will change for sure! Thanks!